.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -44,4 +44,5 @@ Closes #ISSUE_NUMBER.
 ## Checklist
 <!--- Go over all the following points, and make sure you've done anything necessary -->
 - [ ] I have added tests to cover my changes, if necessary.
+- [ ] I have added translations for new strings, if necessary.
 * I have updated the documentation accordingly, if necessary.
```

Comment:
Shall we update this bullet point to be consistent while we're here?

Reply:
Was gonna ask the same 🤣

Reply:
Was there a reason it was a bullet vs checkbox? I was hoping someone would remember :P

Reply:
![](https://i.gyazo.com/6b0f77d1b0aadb8ecbcf9bad547640ab.png)